x/wasm/types/genesis.go

```diff
@@ -47,7 +47,7 @@ func (c Code) ValidateBasic() error {
 	if err := c.CodeInfo.ValidateBasic(); err != nil {
 		return sdkerrors.Wrap(err, "code info")
 	}
-	if err := validateWasmCode(c.CodeBytes); err != nil {
+	if err := validateWasmCode(c.CodeBytes, MaxWasmSize); err != nil {
```

Comment:
This needs to be raised or an import fails for gov contracts
```suggestion
	if err := validateWasmCode(c.CodeBytes, MaxProposalWasmSize); err != nil {
```